scripts/install-chef-suse.sh

```diff
@@ -949,6 +967,8 @@ touch /var/run/crowbar/deploying
 #
 # which exits immediately if /var/run/crowbar/deploying exists.
 
+ensure_crowbar_running
```

Comment:
Hrm, but aren't we creating the "default" proposal for the crowbar barclamp just in the block below?